### ISSUE TITLE
Store assets somewhere else

### ISSUE THREAD

**Issue by Zireael07** [Libre-TrainSim/Libre-TrainSim#404]
### Summary
I tried to download the files (download as zip, not clone). I had to stop after reaching 300 MB...

### Proposal
Store assets separately to the code.

### Alternatives
Lose people because their net can't handle the sizes involved.

**Comment by DasCapschen** [comment]
We could store assets and code separately, but you won't be able to do much with the project if you're missing the assets.  
So you would need to download them anyway. I'm not sure if it makes it better to change where they are at.

But in theory this would not be a problem. We could probably integrate Git LFS with a self-hosted or rented storage server. (for example Hetzner has 1TB storage servers for 4€/month, quite cheap)
What do you think @HaSa1002 ?

**Comment by Zireael07** [comment]
Yes I know, but on a slow and/or rate-limited connection or with a limited HDD space, it's easier to download e.g. 200 and 150 MB twice (e.g. to different partitions) than 350 MB once

**Comment by HaSa1002** [comment]
So the current ZIP Dump by GH is 602 mb in size. Using git lfs won't solve the issue apart from keeping the local .git folder smaller. 

I agree asset management is a topic and a topic which needs a decision because right now, the base game includes a good portion of assets that may be not needed. Ideally, I'd like to provide our assets using the mod system. On the flip side, having these assets allows to clone the repository and start testing immedieatly. And as we can't really load mods in the editor we need some data there to test and verify work.

@Zireael07 I would like to know why you wanted to get the files as GH packaged ZIP file? We have CI builds that have packaged builds and if you plan to play with the source a clone may be a better idea?

I am just a little worried that reducing size for the sake of reducing size will increase the hurdle to work with the project. A game will always require assets to work and simply stripping them away leaves you with something incomplete.

**Comment by Zireael07** [comment]
@HaSa1002 GH packaged zip gets only the latest state, a git clone gets ALL the commit history so even more data and time will get used...

**Comment by HaSa1002** [comment]
But git clone can resume the operation if necessary. Sorry but stripping out the assets of the game for the sake of it generates no value. It's not to say there are no optimisations possible. We simply have to weigh multiple factors. The big default maps are already in their own repositories which causes issues in our current setup. It is not simply done.

While I understand your frustration, I have to keep maintenance in mind. If we'd essentially split the repository even further, the maintenance costs seem to be higher without a real benefit. Perhaps we can provide a stripped down version of the source with releases

**Comment by Zireael07** [comment]
>  Perhaps we can provide a stripped down version of the source with releases

That's a good idea too
